saleor/dashboard/order/forms.py

```diff
@@ -309,23 +319,16 @@ def refund(self):
         return self.try_payment_action(gateway_refund)
 
 
-class VoidPaymentForm(forms.Form):
+class VoidPaymentForm(ManagePaymentForm):
+
+    clean_status = ChargeStatus.NOT_CHARGED
+    clean_error = pgettext_lazy('Payment form error',
+                                'Only pre-authorized payments can be voided')
 
     def __init__(self, *args, **kwargs):
-        self.payment = kwargs.pop('payment')
         super().__init__(*args, **kwargs)
-
-    def clean(self):
-        if self.payment.charge_status != ChargeStatus.NOT_CHARGED:
-            raise forms.ValidationError(
-                pgettext_lazy(
-                    'Payment form error',
-                    'Only pre-authorized payments can be voided'))
-
-    def payment_error(self, message):
-        self.add_error(
-            None, pgettext_lazy(
-                'Payment form error', 'Payment gateway error: %s') % message)
+        self.payment = kwargs.pop('payment')
+        self.fields.pop('amount')
```

Comment:
What is the reason for that?

Reply:
```VoidPaymentForm``` form does not require amount, so we have to pop amount field otherwise it will be rendered in frontend.

Reply:
Maybe we could add a comment so we know why it's there? :+1:

Reply:
Will do. :D